app/routes/screens/race/Race.js

```diff
@@ -63,6 +77,21 @@ class RacePredictor extends Component {
     return "00000".substring(0, 6 - c.length) + c;
   }
 
+  state = {
+    h: 25,
```

Comment:
don't put h  in the state just menuStatus: "collapsed"/ "expand" or something like that